test/User.test.ts

```diff
@@ -58,6 +58,12 @@ describe('User API', () => {
         expect(castedReturn.totalbountyreward).to.equal(245245450245245);
     });
 
+    it('personalstats', async () => {
```

Comment:
Can we update the name of the test? Maybe some like `personalstats with specified stats`. Also, can we add a spot check on the return of the `personalstats` function to make sure the stats we want come back and more importantly a stat we do not want, does not get returned.